### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by Khibar**
Remade web and mobile navbar

### CODE REVIEW

File: src/components/layouts/components/DefaultNavbar.vue [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+        font-style: normal;
+        font-stretch: normal;
+        line-height: normal;
+        letter-spacing: 0.7px;
+        color: #919191;
+        margin: 0px 22px;
+        padding: 0;
+        text-decoration: none
+      }
+
```
Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultNavbar.vue]: Extra line.

File: src/components/layouts/default.vue [hunk 2]
```diff
@@ -1,89 +1,10 @@
...
-                >
-                  {{ t('navigation.signUp') }}
-                </el-menu-item>
-              </template>
-            </el-menu>
-          </el-row>
-        </el-col>
-      </el-row>
+    <el-header
+      height="48px"
```
Comment (IlyaDonskikh) [inline src/components/layouts/default.vue]: Why do not set the property for `DefaultNavbar `?

File: src/components/layouts/default.vue [hunk 3]
```diff
@@ -128,11 +39,12 @@ export default {
   flex-direction: column;
 
   &__header {
-    position: relative;
+    position: fixed;
```
Comment (IlyaDonskikh) [inline src/components/layouts/default.vue]: `relative` is correct option.

File: src/locales/en/components/layouts/default.js [hunk 4]
```diff
@@ -1,14 +1,5 @@
...
-      home: 'Home',
-      signIn: 'Sign In',
-      signOut: 'Sign Out',
-      signUp: 'Sign Up',
-      user: 'Hello %{name}',
-      properties: 'Properties',
-      propertyOwnerPanel: 'PO-Dashboard',
-      dashboard: 'Dashboard'
-    }
+
```
Comment (IlyaDonskikh) [inline src/locales/en/components/layouts/default.js]: Extra line.

File: src/locales/en/components/layouts/components/DefaultNavbar.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export default {
+  DefaultNavbar: {
+    navigation: {
+      home: 'Home',
+      signIn: 'Sign In',
+      signOut: 'Sign Out',
+      signUp: 'Sign Up',
+      user: 'Hello %{name}',
```
Comment (IlyaDonskikh) [inline src/locales/en/components/layouts/components/DefaultNavbar.js]: Leave only `user: '%{name}'`.

File: src/components/layouts/components/DefaultNavbar.vue [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+            class="default-navbar-web-menu__link"
+          >
+            {{ t('navigation.propertyOwnerPanel') }}
+          </router-link>
+        </template>
+      </div>
+      <div class="default-navbar-web-auth">
+        <template v-if="isAuthenticated">
+          <i class="fas fa-user default-navbar-web-auth__user-icon" />
+          <h5
```
Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultNavbar.vue]: Why do not user elements `dropdown` menu? And using `h` elements in navigation is a bad idea. 

You have to user `h` to mark the titles.

File: src/components/layouts/default.vue (hunk 3 above)
Comment (Khibar) [reply src/components/layouts/default.vue]: are you sure bro, if use relative, the navigation will disappear when user scroll down, please just confirm which behaviour you want,

File: src/components/layouts/components/DefaultNavbar.vue (hunk 1 above)
Comment (Khibar) [reply src/components/layouts/components/DefaultNavbar.vue]: done: 4b97971e2904b95d5e1a9a05e002469b2d15284a

File: src/components/layouts/default.vue (hunk 2 above)
Comment (Khibar) [reply src/components/layouts/default.vue]: moved to css: 4b97971e2904b95d5e1a9a05e002469b2d15284a

File: src/components/layouts/default.vue (hunk 3 above)
Comment (Khibar) [reply src/components/layouts/default.vue]: changed to relative: 4b97971e2904b95d5e1a9a05e002469b2d15284a

File: src/locales/en/components/layouts/default.js (hunk 4 above)
Comment (Khibar) [reply src/locales/en/components/layouts/default.js]: removed lines: 4b97971e2904b95d5e1a9a05e002469b2d15284a

File: src/locales/en/components/layouts/components/DefaultNavbar.js (hunk 5 above)
Comment (Khibar) [reply src/locales/en/components/layouts/components/DefaultNavbar.js]: removed hello: 4b97971e2904b95d5e1a9a05e002469b2d15284a

File: src/components/layouts/components/DefaultNavbar.vue (hunk 6 above)
Comment (Khibar) [reply src/components/layouts/components/DefaultNavbar.vue]: changed all h5 to div: 4b97971e2904b95d5e1a9a05e002469b2d15284a

**Pull request merged**